test/Interop/C/swiftify-import/clang-includes-noexport.swift

```diff
@@ -0,0 +1,60 @@
+// REQUIRES: swift_feature_SafeInteropWrappers
+
+// RUN: %target-swift-ide-test -print-module -module-to-print=ClangIncludesNoExportModule -plugin-path %swift-plugin-dir -I %S/Inputs -source-filename=x -enable-experimental-feature SafeInteropWrappers | %FileCheck %s
+
+// swift-ide-test doesn't currently typecheck the macro expansions, so run the compiler as well
+// RUN: %target-swift-frontend -emit-module -plugin-path %swift-plugin-dir -o %t/ClangIncludesNoExport.swiftmodule -I %S/Inputs -enable-experimental-feature SafeInteropWrappers %s
+
+import ClangIncludesNoExportModule
+
+// CHECK:      import ModuleA
+// CHECK-NEXT: import ModuleB
+// CHECK-NOT:  import
```

Comment:
@DougGregor I don't fully get why only the module's imports that themselves are top-level modules show up in the dump. It seems to work regardless, but it seems a bit off. Do you know if this is intended?

Reply:
Somewhat fixed by https://github.com/swiftlang/swift/pull/81859/commits/a56810277defa28d8c950bec1915359ecd56ecd6
@bnbarham I think this affects IDE stuff, so I didn't dare make it actually print every module even though it would've been kinda nice if this reflected the full imports. Do you have more context for why this is the way it is?